src/pushover_complete/pushover_api.py

```diff
@@ -136,10 +140,14 @@ def _send_message(self, user, message, device=None, title=None, url=None, url_ti
             'sound': sound,
             'html': html
         }
-
+        
+        if image_path is not None and os.path.isfile(image_path):
+            with open(image_path, 'rb') as f:
+                file = {'attachment': (image_path, f)}
```

Comment:
I see in the `requests` docs that you can use a "file tuple" but it's not explained why the filename could be important. Do we need `(image_path, f)` or do we just want `'attachment': f`? The other thing to check is if `image_path` is a relative or absolute path, does `requests` want/deal with the full `/Users/foo/whatever.png` or does it really want just `whatever.png`? If the latter we should do something with `os.path` to extract just the part we want.

Reply:
I assume the file name is used to specify a different name for downloading, but it doesn't look like Pushover is using this for anything; it is required in the case of the tuple. I suppose the tuple could be avoided and just pass in the file object.

Alternatively, `os.path.split(image_path)[1]` would be a way to just pass in the file name.

I like the idea of dropping the tuple to be honest.